Functions/xASL_init_FileSystem.m

```diff
@@ -186,6 +186,7 @@
 x.P.Atlas.Tatu_ICA_PCA                      = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'TatuICA_PCA.nii.gz');
 x.P.Atlas.Tatu_ICA_L_ICA_R_PCA              = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'LabelingTerritories.nii.gz');
 x.P.Atlas.Tatu_ACA_MCA_PCA_Prox_Med_Dist    = fullfile(x.D.MapsSPMmodifiedDir, 'VascularTerritories', 'ATTbasedFlowTerritories.nii.gz');
+x.P.Atlas.DKT                  = fullfile(x.D.AtlasDir, 'Desikan_Killiany_MNI_SPM12.nii');
```

Comment:
This shouldn't be necessary. If the files are in the atlas directory, they should be found automatically.

Reply:
Okay but how do we tell xASL (using the datapar) that we want to use this atlas?

Reply:
It uses the filename. So if the atlas is called `Desikan_Killiany_MNI_SPM12` you just have to use `Desikan_Killiany_MNI_SPM12` in the `dataPar.json`.

Reply:
We only do it differently for the atlases that are located in the SPM modified directory. We probably can't just move them, even though that would be my preferred solution.

Reply:
Okay!